airflow/providers/google/cloud/example_dags/example_mssql_to_bigquery.py

```diff
@@ -0,0 +1,134 @@
+#
```

Comment:
The example dag should be written as a system test.
Since we already have one 
https://github.com/apache/airflow/blob/main/tests/system/providers/google/cloud/gcs/example_mssql_to_gcs.py
maybe you can modify it to complete what is missing?

Reply:
After a second thought, my example is too specific for solving the original issue - so currently I don't see a point in expanding the existing example, as nothing is really missing.

Reply:
can you try to insert this value `4.1000000000000005` to the sql table and try to export it to BQ

Reply:
> can you try to insert this value `4.1000000000000005` to the sql table and try to export it to BQ

It's been a year since I've been working on this PR - if you recognize that there is a new related issue, please open a new issue for that.